File: flutter_local_notifications/ios/flutter_local_notifications/.build/arm64-apple-macosx/debug/flutter_local_notifications.build/DerivedSources/resource_bundle_accessor.m
#import <Foundation/Foundation.h>

NSBundle* flutter_local_notifications_SWIFTPM_MODULE_BUNDLE() {
    NSURL *bundleURL = [[[NSBundle mainBundle] bundleURL] URLByAppendingPathComponent:@"flutter_local_notifications_flutter_local_notifications.bundle"];

    NSBundle *preferredBundle = [NSBundle bundleWithURL:bundleURL];
    if (preferredBundle == nil) {
      return [NSBundle bundleWithPath:@"/Users/omarkhan/flutter_local_notifications/flutter_local_notifications/ios/flutter_local_notifications/.build/arm64-apple-macosx/debug/flutter_local_notifications_flutter_local_notifications.bundle"];
    }

    return preferredBundle;
}

File: flutter_local_notifications/macos/flutter_local_notifications/.build/arm64-apple-macosx/debug/flutter_local_notifications.build/DerivedSources/resource_bundle_accessor.swift
import Foundation

extension Foundation.Bundle {
    static let module: Bundle = {
        let mainPath = Bundle.main.bundleURL.appendingPathComponent("flutter_local_notifications_flutter_local_notifications.bundle").path
        let buildPath = "/Users/omarkhan/flutter_local_notifications/flutter_local_notifications/macos/flutter_local_notifications/.build/arm64-apple-macosx/debug/flutter_local_notifications_flutter_local_notifications.bundle"

        let preferredBundle = Bundle(path: mainPath)

        guard let bundle = preferredBundle ?? Bundle(path: buildPath) else {
            // Users can write a function called fatalError themselves, we should be resilient against that.
            Swift.fatalError("could not load resource bundle: from \(mainPath) or \(buildPath)")
        }

        return bundle
    }()
}